openslides_backend/action/actions/motion_submitter/create.py

```diff
@@ -31,9 +31,6 @@ class MotionSubmitterCreateAction(CreateActionWithInferredMeetingMixin, CreateAc
     relation_field_for_meeting = "motion_id"
 
     def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
-        """
-        Check if motion and user belong to the same meeting if the user is a temporary user.
```

Comment:
The docstring can stay, since it now applies to all users instead of only temporary ones.

Reply:
Rm the guest_meeting_ids.

Reply:
Add a comment: _Check if motion and user belong to the same meeting._